packages/boxel-ui/addon/components/field-container/index.gts

```diff
@@ -51,6 +51,67 @@ const FieldContainer: TemplateOnlyComponent<Signature> = <template>
       {{/if}}
     </Tag>
   {{/let}}
+
+  <style>
+    .boxel-field {
```

Comment:
I agree that the variable element name means it's better to keep using this class.